supabase/functions/backend/misc/utils.ts

```diff
@@ -1,17 +1,17 @@
 export const intervalToString = (interval: string) => {
-	const [hours, minutes, seconds] = interval.split(':').map(Number)
+  const [hours, minutes, seconds] = interval.split(':').map(Number)
 
-	if (hours > 0) {
-		return `${hours} ${hours === 1 ? 'hour' : 'hours'}`
-	} else if (minutes > 0) {
-		return `${minutes} ${minutes === 1 ? 'minute' : 'minutes'}`
-	} else if (seconds > 0) {
-		return `${seconds} ${seconds === 1 ? 'second' : 'seconds'}`
-	} else {
-		return 'Invalid interval format'
-	}
+  if (hours > 0) {
+    return `${hours} ${hours === 1 ? 'hour' : 'hours'}`
+  } else if (minutes > 0) {
+    return `${minutes} ${minutes === 1 ? 'minute' : 'minutes'}`
+  } else if (seconds > 0) {
+    return `${seconds} ${seconds === 1 ? 'second' : 'seconds'}`
+  } else {
+    return 'Invalid interval format'
+  }
```

Comment:
You haven't considered `interval` that includes both `hours` and `minutes` (or `seconds`)